utils/versions.py

```diff
@@ -75,29 +61,17 @@ def get_most_recent_vernacular_version(collection_id: Union[int, str]):
     recent_version = sorted(versions)[-1]
     return f"{collection_id}/{recent_version}/"
 
-def get_pages(version: str, pipeline_step: str, **kwargs):
-    if pipeline_step == 'vernacular': 
-        data_root = os.environ.get("VERNACULAR_DATA", "file:///tmp")
-    elif pipeline_step == 'mapped':
-        data_root = os.environ.get("MAPPED_DATA", "file:///tmp")
-    else:
-        raise Exception("Invalid pipeline step")
-
+def get_vernacular_page(version, **kwargs):
```

Comment:
I think this should be `get_vernacular_pages`?

Reply:
Hey @barbarahui - are you familiar with suggestions? You can make one line changes like this as a suggestion in your PR response. It's an option to the left of H B I (heading, bold, italics) in this comment box.

Reply:
```suggestion
def get_vernacular_pages(version, **kwargs):
```

Reply:
Ah! Will def use suggestions in future!